src/main/java/com/gk/portfolio/models/FeedbackModel.java

```diff
@@ -0,0 +1,19 @@
+package com.gk.portfolio.models;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+public class FeedbackModel {
+
+    @NotBlank
+    public String name;
+    @NotBlank
+    public String position;
+
+    @Min(1)
```

Comment:
take care about Json property names (it is preferable to have not-camel case names for API)